packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -443,6 +497,10 @@ describe('validation', function() {
 
       it('linked-resource-missing-linkName');
 
+      it.skip('linked-resource-missing-versionTag');
```

Comment:
These tests currently fail, as the `bindingType` implementation for `linkedResources` isn't very strict. In actuality an what's defined in the fixtures should probably cause a validation error.